pkg/jobrunner/controller.go

```diff
@@ -165,6 +197,57 @@ func (ctrl *JobController) Run(ctx context.Context, task Job, cleanup bool) (*Jo
 	return result, nil
 }
 
+func getcrd(config *rest.Config, group string, ver string, kind string, namespace string, name string) batchv1.JobSpec {
+	// Get the CRD instance
+	log.Printf("> Getting custom resource instance %s.%s/%s/%s", group, kind, namespace, name)
```

Comment:
formattið a strengnum  er skrítið ... apiVersion er venjulega group/ver eða isbank.is/v1